spec/features/access_data_flow_spec.rb

```diff
@@ -39,25 +39,109 @@
 
   scenario 'shows download and API options correctly' do
     visit('/registers/country')
-
     expect(page).to have_css '.highlight-box', count: 2
-
     expect(page).to have_content('Download the data')
     expect(page).to have_content('Use the API')
   end
 
+  scenario 'the states for all pages in API flow are 200' do
+    visit('/registers/country')
+    expect(page.status_code).to eq(200)
+
+    click_link('Use the API')
+    expect(page.status_code).to eq(200)
+
+    click_link('Skip this step')
+    expect(page.status_code).to eq(200)
+  end
+
+  scenario 'the states for all pages in download flow are 200' do
+    visit('/registers/country')
+    expect(page.status_code).to eq(200)
+
+    click_link('Download the data')
+    expect(page.status_code).to eq(200)
+
+    click_link('Skip this step')
+    expect(page.status_code).to eq(200)
+  end
+
   scenario 'goes to questionnaire correctly when download option chosen' do
     visit('/registers/country')
     click_link('Download the data')
 
     expect(page).to have_content('Before you download the data')
     expect(page).to have_content('Help us improve GOV.UK Registers')
+
+    expect(page).to have_content('What part of government are you working for?')
```

Comment:
I think we should be able to use `include` and make an array with all the expected values: https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/include-matcher